### ISSUE TITLE
Issue packaging with pyinstaller

### ISSUE THREAD

**Issue by kiyros** [ocrmypdf/OCRmyPDF#1024]
below is my `fileToOcr` function
```python
os.environ["TESSDATA_PREFIX"] = "Location of Tessdata Folder"
if __name__ == '__main__':
        start = timer()
        ocrmypdf.ocr(pdf, output_file=f'{outputPath}\\{pdfOriginalName.strip(".pdf").strip(".PDF")}.PDF',
        redo_ocr=True, output_type="pdf")
        end = timer()

        print(end - start, " Seconds")

        # Remove checked pdf from queue/check Folder
        os.remove(pdf)
```

This code runs fine inside visual studio but as soon as i package using pyinstaller
`pyinstaller --onefile  --collect-data pikepdf --hidden-import pikepdf --copy-metadata pikepdf --collect-data ocrmypdf --hidden-import ocrmypdf --copy-metadata ocrmypdf -w 'dailyEDI_PDF_checker.py'`

it gives me this error when i run the .EXE
``` 
  File "ocrmypdf\api.py", line 339, in ocr
  File "ocrmypdf\_validation.py", line 246, in check_options
  File "ocrmypdf\_validation.py", line 240, in _check_plugin_options
AttributeError: 'NoneType' object has no attribute 'languages'

```

I am not entirely sure why it is giving me 'noneType' 

OS: Windows
installed using: Pip
Expected behavior: To run the ocr Function smoothly

**Comment by jbarlow83** [comment]
I believe you need to --hidden-import all of ocrmypdf's built in plugins because PyInstaller doesn't understand the plugin manager.

I did try creating a PyInstaller edition of ocrmypdf before but got worried about the time to maintain another build artifact.

**Comment by kiyros** [comment]
By ocrmypdf's "Hidden imports" do you mean ghostscript, pngquant etc?

**Comment by jbarlow83** [comment]
No, I believe you need to specify `--hidden-import ocrmypdf.builtin_plugins`. Something along the 

I don't know of any way to use PyInstaller to bundle third party dependencies like Tesseract and Ghostscript, which have their own installers and license agreements.

**Comment by yangyunlv** [comment]
I also encountered the same problem 
use --hidden-import ocrmypdf.builtin_plugins  and --copy-metadata  Even if they cannot solve the problem, the same error will still occur

**Comment by mathieugruson** [comment]
Anyone has find a solution on this topic ?
